pkg/network/render.go

```diff
@@ -41,6 +48,7 @@ func Validate(conf *netv1.NetworkConfigSpec) error {
 	errs := []error{}
 
 	errs = append(errs, ValidateDefaultNetwork(conf)...)
+	errs = append(errs, ValidateMultus(conf)...)
```

Comment:
Shall we also validate additional networks here? If it's not provided, validation of additional network returns immediately.

Reply:
@zshi-redhat no, the validate of additional networks would be handled by the additional network rendering, not multus rendering. Multus and AdditionalNetworks are completely separate codepaths that do not depend on each other.